cluster/saltbase/salt/atop/init.sls

```diff
@@ -0,0 +1,15 @@
+{% if grains['os_family'] != 'RedHat' %}
+
+atop:
+  pkg:
+    - installed
```

Comment:
Ah, right. I should know what's going on, I built this image. Since all the deps are built in now, we actually haven't had to do an apt-get update yet. It finds atop after that.

Please file an internal bug to get this added to the next ContainerVM rev. cc @dchen1107 (it's an easy add to the manifest)